openfl-tutorials/experimental/Workflow_Interface_104_MNIST_XPU.ipynb

```diff
@@ -0,0 +1,729 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "14821d97",
+   "metadata": {},
+   "source": [
+    "# Workflow Interface 104: MNIST XPU\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/intel/openfl/blob/develop/openfl-tutorials/experimental/Workflow_Interface_104_MNIST_XPU.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bd059520",
+   "metadata": {},
+   "source": [
+    "Welcome to the OpenFL Experimental Workflow Interface tutorial using XPU! This notebook introduces the API to get up and running with your first horizontal federated learning workflow using a XPU. This work has the following goals:\n",
```

Comment:
It would help to replace XPU with Intel® Data Center GPU Max Series at the beginning of the notebook, and establish that XPU refers to this when you select the device in IPEX. Otherwise, people may be left confused wondering what an XPU is

Reply:
Added an explanation of what is a XPU and to what refers when select 'xpu' device on IPEX